examples/with-http2/server.js

```diff
@@ -6,7 +6,13 @@ const port = parseInt(process.env.PORT, 10) || 3000
 const dev = process.env.NODE_ENV !== 'production'
 
 // Init the Next app:
-const app = next({ dev })
+const app = next({
+  dev,
+  conf: {
+    // this needs to be set false  unitl to a bug in the compression npm module ets fixed.
+    compress: false,
```

Comment:
This should be set in `next.config.js`, not here!

Reply:
yes of course!! moved it to `next.config.js`